src/StubFileFetcher.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare( strict_types=1 );
+
+namespace FileFetcher;
+
+/**
+ * @since 4.1
+ *
+ * @licence GNU GPL v2+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class NullFileFetcher implements FileFetcher {
```

Comment:
:D ... yea, so all code not tested is broken, even if it just is a NullObject >_>